pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -140,6 +140,22 @@ public void createPartitionedTopic(@PathParam("tenant") String tenant, @PathPara
         internalCreatePartitionedTopic(numPartitions, authoritative);
     }
 
+    @PUT
+    @Path("/{tenant}/{namespace}/{topic}/nonpartitioned")
```

Comment:
What about just /{tenant}/{namespace}/{topic} for the endpoint to create? It seems more natural and the /nonpartitioned is kind of confusing and seems inconsistent with the rest of the endpoints for topics. For example we do /{tenant}/{namespace}/{topic}/stats to get the stats and not /{tenant}/{namespace}/{topic}/nonpartitioned-stats